components/Modal.js

```diff
@@ -18,7 +18,7 @@ export default function Modal({ show = false, setShow, title, children }) {
           <div className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" />
         </Transition.Child>
 
-        <div className="fixed inset-0 z-10 overflow-y-auto">
+        <div className="w-fit m-auto fixed inset-0 z-10 overflow-y-auto">

```

Comment:
I think this has broken the model for other areas, this is an example for the Playground page

![Uploading Screenshot 2023-06-21 at 17.01.58.png…]()

Reply:
I added a modalStyles prop for this. It will modify the styles only if the prop is passed otherwise default. We needed this because the share modal wasn't looking good without 'w-fit'.